### ISSUE TITLE
Do not vertically align text in tables from Rd files

### ISSUE THREAD

**Issue by etiennebacher** [etiennebacher/altdoc#265]
https://grantmcdermott.com/tinyplot/man/tpar.html#additional-graphical-parameters

I'd rather have the arg names not vertically centered, it makes it harder to distinguish to which argument the text corresponds when there are big paragraphs:

![image](https://github.com/etiennebacher/altdoc/assets/52219252/57d12e91-034d-4748-8827-54da6e146266)

Not sure what role `altdoc` has in this (and it's very low priority) but I wanted to store this somewhere

**Comment by vincentarelbundock** [comment]
The intention is to top-align, and it seems to work here: https://marginaleffects.com/man/predictions.html

This is the line in altdoc which is responsible for this: https://github.com/etiennebacher/altdoc/blob/main/R/rd2qmd.R#L22

I'm not sure what is going on. Maybe there a semicolon missing? Not sure...

**Comment by vincentarelbundock** [comment]
This does not appear to be an altdoc problem. The issue is that @grantmcdermott is using a special tabular environment. A better way to handle this would be to enable markdown formatting in roxygen2, and to put the extra parameters in a markdown/ASCII table (ex: via tinytable)

**Comment by vincentarelbundock** [comment]
Yeah, pretty sure this was a `tinyplot` weirdness. Closing now, but feel free to re-open if relevant.

**Comment by etiennebacher** [comment]
Thanks for cleaning up existing issues, Vincent!
